src/Core/Picking.js

```diff
@@ -208,8 +208,10 @@ export default {
         } else {
             raycaster.layers.enableAll();
         }
+        // Raycaster use NDC coordinate
+        const normalized = new THREE.Vector2();
```

Comment:
Could you instantiate this `THREE.Vector2()` outside of the method (the same way as `raycaster` constant is implemented in line 100 for instance) ? If I got it wright, implementing a new constant at each call of a method implies that the garbage collector is used to clear memory space allocated to each constant, which takes a bit of time.

Reply:
Done :)